packages/web/src/pages/stats/components/bulletin-card.tsx

```diff
@@ -10,16 +10,28 @@ interface IProps {
   bodyStyle: Record<string, any>;
 }
 
+
 const BulletinCard: React.FC<IProps> = ({ data, type, race, bodyStyle }) => {
   const bulletinData = data["intelBulletins"][race as "soviet"];
+  const [toggle, setToggle] = useState(true)
 
   // Here are all the bulletin data
   // console.log(bulletinData)
 
+  const sliced = Object.fromEntries(
+      Object.entries(bulletinData).slice(0, 20)
```

Comment:
This is good but first we need to sort the data. It's unsorted. Please take a look how the data object look slike and sort it based on the values. 

I think something like this could work :
```
sort((a, b) => {
    if (a.value < b.value) {
      return -1;
    }
    if (a.value > b.value) {
      return 1;
    }
    return 0;
  });
  ``